notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<h2>Check explanations and baselines</h2><p> </p><p>To better make sense of your feature attributions, you can compare them with your model's baseline. For image models, the <code>baseline_score</code> returned by AI Explanations is the score your model would give an image input with the baseline we specified. The baseline is different for each class in the model. Every time your model predicts <code>tulip</code> as the top class, you'll see the same baseline score.</p><p> </p><p>Earlier, you used a baseline image of <code>np.random</code> randomly generated values. If you'd like the baseline for your model to be solid black and white images instead, pass <code>[0,1]</code> as the value to <code>input_baselines</code> in your <code>explanation_metadata.json</code> file above. If the <code>baseline_score</code> is very close to the value of <code>example_score</code>, the highlighted pixels may not be meaningful.</p><p> </p><p>Calculate the difference between <code>baseline_score</code> and <code>example_score</code> for the three test images above.</p><p>Note that the score values for classification models are <em>probabilities</em>: the confidence your model has in its predicted class. A score of 0.90 for "tulip" means your model has classified the image as a tulip with 90% confidence.</p><p> </p><p>The code below checks baselines for the IG model. To inspect your XRAI model, swap out the <code>ig_response</code> and <code>IG_VERSION</code> variables below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>